test/evergreen.yml

```diff
@@ -952,6 +955,23 @@ tasks:
             set -o verbose
             UBSAN_OPTIONS=print_stacktrace=1:halt_on_error=1:abort_on_error=1:disable_coredump=0 ./ex_access
 
+  # Start of cppsuite test tasks.
+
+  - name: poc-test
```

Comment:
Consider including "cpp" as part of the task name, which makes the new task more self-explained.